defaults/main.yml

```diff
@@ -7,9 +7,15 @@ patroni_config_dir: /etc/patroni
 patroni_system_user: postgres
 patroni_system_group: postgres
 
+patroni_el_packages:

```

Comment:
`patroni_el_packages` isn't needed here.
I would move `patroni_system_packages` variable from defaults/main.yaml to vars/Debian.yml and vars/RedHat.yml files and then extend `patroni_system_packages` in RedHat.yml with packages which you specified in `patroni_el_packages`